src/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+include(GNUInstallDirs)
+
+option(EKAT_AUTO_ENABLE_DEPENDENCIES  "Automatically enable EKAT packages if needed" OFF)
+
+option(EKAT_ENABLE_ALGORITHM  "Enable EKAT algorithm utilities" OFF)
+option(EKAT_ENABLE_KOKKOS     "Enable EKAT kokkos utilities" OFF)
+option(EKAT_ENABLE_LOGGING    "Enable EKAT spdlog utilities" OFF)
+option(EKAT_ENABLE_PACK       "Enable EKAT packs utilities" OFF)
+option(EKAT_ENABLE_PARSER     "Enable EKAT yaml parsing utilities" OFF)
```

Comment:
I called the module "Parser" in case we later decide to add support for parsing other stuff (e.g., XML).

Reply:
That's a fair point. Maybe I can just roll back to `EKAT_ENABLE_YAML_PARSER` (which was my original option).

Reply:
LoL, indeed, in the test folder I still had `if (EKAT_ENABLE_YAML_PARSER)`, so now we had a mix. I restored `EKAT_ENABLE_YAML_PARSER` in the src folder as well.